.coveragerc

```diff
@@ -10,3 +10,5 @@ omit =
 
 [report]
 show_missing = True
+precision = 2
+fail_under = 96.32
```

Comment:
Where did this number come from?

Reply:
I'm jumping in unbidden here, but IIRC, this was explained as being the current code coverage for the project—set as a line to never slip below again (i.e. drawing a snapshot line-in-the-sand).

Reply:
I'm just gonna raise this here but I don't see it as a blocker. I've seen code coverage drop when for example you remove lines of code even though the changes you made are 100% tested (because the number of lines of covered code is now slightly smaller than the number of uncovered lines) so my concern is this arbitrary number may cause problems later down the road if that happens but I'm fine addressing that then.

Reply:
@hmstepanek Remember that we can always, as humans, choose to ignore a failing CI (e.g. code coverage) and merge anyway. So I tend to feel like this issue is surmountable by using common sense.

Reply:
With this approach you actually can't just ignore a failing `make coverage` and merge it, as that would mean that `make coverage` would be failing on master. But you _can_ just lower the `fail_under` number in `.coveragerc` and commit that on your branch, that's perfectly fine.

Our longer term aim, which will fix this and other issues, is to get coverage to 100% and set `fail_under = 100.00`. Even if we have to make liberal use of `nocover` comments and files omitted from coverage to do so (although with >96% we're pretty close already). With coverage pinned at 100% removing covered lines won't reduce the coverage. It'll also mean that all new lines of code have to either have 100% coverage or you can use `nocover` comments and omitted files (both of which we intend to make plenty of use of)

Reply:
You can also take it as an opportunity to make the universe a better place and up the coverage, or hunt for lines to skip etc.